app/assets/stylesheets/explosions.scss

```diff
@@ -0,0 +1,44 @@
+@import "_variables";
+
+.explode {
+  animation: bgColor 1s infinite linear;
+  animation-delay: var(--count);
+}
+
+.tile {
+ display: block;
+ float: left;
+ width:30px;
+ height:30px; 
+}
+
+.line {
+  height:6px;
+}
+
+@keyframes bgColor {
+  12.5% {
+    background-color: #FF0000;
+  }
+  25% {
+    background-color: #FFA500;
+  }
+  37.5% {
+    background-color: #FFFF00;
+  }
+  50% {
+    background-color: #7FFF00;
+  }
+  62.5% {
+    background-color: #00FFFF;
+  }
+  75% {
+    background-color: #0000FF;
+  }
+  87.5% {
+    background-color: #9932CC;
+  }
+  100% {
+    background-color: #FF1493;
+  }
+}
```

Comment:
Smart!

Reply:
yeah, wow. i had no idea you could do so much with css :D

Reply:
I got this from a tutorial, CSS animations are great.